packages/test-runner/src/reporter.ts

```diff
@@ -25,4 +25,5 @@ export interface Reporter {
   onTestEnd(test: Test, result: TestResult): void;
   onTimeout(timeout: number): void;
   onEnd(): void;
+  onLoadingFailure(file: string, error: any): void;
```

Comment:
Since this is about traversal, parsing and loading, should we call it simply `onError` / `onFatalError` / `onFailure`?

Reply:
I always got confused about onError vs onTestError. How about `onStartupError`?

Reply:
We have onBegin/onEnd and onTestBegin/onTestEnd, so I think onError will be taken well. onParseError? :) Anyways, doesn't matter, we'll rename it.